lib/ansible/galaxy/collection.py

```diff
@@ -235,6 +283,30 @@ def install(self, path, b_temp_path):
 
             raise
 
+    def install_scm(self, b_collection_output_path):
+        """
+        Generates the Ansible collection artifact data from a galaxy.yml and installs the artifact to a directory.
+        This should follow the same pattern as build_collection, but instead of creating an artifact, install it.
+        :param b_collection_output_path: The installation directory for the collection artifact.
+        """
```

Comment:
```suggestion
        """Install collection from source control into given dir.

        Generates the Ansible collection artifact data from a galaxy.yml and installs
         the artifact to a directory.
        This should follow the same pattern as build_collection, but instead of creating
        an artifact, install it.

        :param b_collection_output_path: The installation directory for the collection artifact.
        """
```